docs/manual/lore/faction-proteron.md

```diff
@@ -0,0 +1,61 @@
+# Sovereign Proteron Autarchy
+
+Formed from the former Great House Proteron, the Sovereign Proteron Autarchy played a pivotal role in the history of the universe.
+It arose from an Emperor's desire to create a revolution in the sphere of society and societal structure.
+The project performed all too well, creating a Great House that surpassed its progenitor in nearly every way, and chafing at its restraints.
+These tensions escalated into all-out war, with devastating consequences to both bodies, because neither have the Proteron heard of restraint nor the Emperor of caution.
+
+## The Facts
+
+* Leader: High Autarch Python
+* Leading Structure: Circle of Autarchs
+* Government: Autarchy
+* Formation: UST 47
+* Homeworld: <!-- TODO: find the actual homeworld-->
+* Important Figures:
+    * Autarchs
+      * HA Python
+
+### House Proteron Society
+
+House Proteron worlds were governed by a totalitarian regime that required its citizens to follow their dictated daily routines, severely punishing transgressions under a zero-tolerance policy. A Proteron citizen was never their own person. They were an asset of the government, their life's labor neatly represented in a book keeping column somewhere. Even the details of their private life were managed and tallied by the authorities. Despite the lack of freedom and privacy, Proteron citizens were not slaves. The government kept them in top shape, providing an adequate supply of health care, relaxation, and entertainment, making sure all predispositions and tastes were catered for. This kept citizens happy to trade independence and privacy for comfort and security. As such, a subject was often able and willing to work at optimal performance. A Proteron citizen's life was thus quite a satisfying one, though excitement otherwise than the terminal kind was rare.
+
+House Proteron's worlds were all specialist worlds. They were geared to performing a limited selection of tasks according to the planet's natural properties or location. The inhabitants on a world were placed there by the government. Each individual was measured and tested, then sent to the world where they could be of the most use. As such, Proteron society was built of individuals rather than families - a family would only have complicated the relocation process.
+
+The local planetary governments were directly responsible to an Autarch. An Autarch managed many worlds as a sector, with the number depending on how densely populated and productive the worlds were. Once every four years all the Proteron Autarchs met in a Circle. During these meeting, the Autarchs would evaluate the development of each Proteron world individually as well as that of the greater Proteron body in general. Based on this evaluation, goals were set for the next four years. Each Autarch was expected to meet or exceed these goals, and those Autarchs who failed to do so were removed from office - in a very permanent manner.
```

Comment:
years -> cycles

Reply:
Ah, fine. Is it standard to mention only cycles? The original had years, I left it as is

Reply:
It's a bit of a mess. However, to be consistent, in general all documents refer to cycles, periods, and seconds, which roughly align with "years", "days", and seconds.